decidim-core/app/views/decidim/notifications_settings/show.html.erb

```diff
@@ -55,6 +55,20 @@
           <span class="switch-label"><%= t(".email_on_moderations") %></span>
         </label>
       </div>
+
+      <% Decidim.component_notification_settings_registry.manifests.filter{ |a| a.settings_area == :administrators }.each do |manifest| %>
```

Comment:
This needs to be `Decidim.notification_settings_registry` after the changes.

The notification settings page throws an error when you enter that page now.

Could you also add a test that checks the user can at least enter the notification settings page please? :pray:

Reply:
In decidim-core/spec/system/account_spec.rb is a context defined where it tries to access notification settings path
`    context "when on the notifications settings page" do
      before do
        visit decidim.notifications_settings_path
      end

      it "updates the user's notifications" do
        within ".switch.newsletter_notifications" do
          page.find(".switch-paddle").click
        end

        within "form.edit_user" do
          find("*[type=submit]").click
        end

        within_flash_messages do
          expect(page).to have_content("successfully")
        end
      end
    end`